lib/puppet/face/strings.rb

```diff
@@ -146,9 +142,9 @@
   # Checks that the required features are installed.
   # @return [void]
   def check_required_features
-    raise RuntimeError, "The 'yard' gem must be installed in order to use this face." unless Puppet.features.yard?
-    raise RuntimeError, "The 'rgen' gem must be installed in order to use this face." unless Puppet.features.rgen?
-    raise RuntimeError, 'This face requires Ruby 1.9 or greater.' if RUBY_VERSION.match?(/^1\.8/)
+    raise "The 'yard' gem must be installed in order to use this face." unless Puppet.features.yard?
+    raise "The 'rgen' gem must be installed in order to use this face." unless Puppet.features.rgen?
+    raise 'This face requires Ruby 1.9 or greater.' if RUBY_VERSION.match?(%r{^1\.8})
```

Comment:
The funny thing here is that `.match?` didn’t exist until Ruby 2.4.6.

Reply:
hmm yes.. and to be honest given the minimum ruby for this gem is now 2.5x i wonder if these constraints could be removed?